client/rest-high-level/src/main/java/org/elasticsearch/client/IndexLifecycleClient.java

```diff
@@ -139,6 +140,34 @@ public AcknowledgedResponse stopILM(StopILMRequest request, RequestOptions optio
                 AcknowledgedResponse::fromXContent, emptySet());
     }
 
+    /**
+     * Get the status of index lifecycle management
+     * See <a href="https://fix-me-when-we-have-docs.com">
+     * the docs</a> for more.
+     *
+     * @param request the request with user defined timeouts.
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     */
+    public StatusILMResponse StatusILM(TimedRequest request, RequestOptions options) throws IOException {
```

Comment:
Method names should start with a lowercase letter, so this should be `statusILM`.

Or maybe `getILMStatus`? `statusILM` doesn't make much sense seeing it for the first time - there's no verb there.